common/src/main/scala/explore/components/state/LogoutTracker.scala

```diff
@@ -64,10 +64,9 @@ object LogoutTracker {
           bc
         }.flatMap(bc => $.modState(State.bc.replace(bc.some)))
       }
-      .componentWillUnmount { $ =>
-        implicit val ctx = $.props.ctx
-        $.state.bc.map(bc => IO(bc.close()).attempt.void).orEmpty.runAsyncAndForget
-      }
+      .componentWillUnmount(
+        _.state.bc.map(bc => IO(bc.close()).attempt.void).orEmpty
```

Comment:
No need to `runAsyncAndForget`?

Reply:
With the bundle we use, `IO` is a recognized as a first-class citizen in `scalajs-react` and can be returned in lifecycle methods instead of `Callback` (except `shouldComponentUpdate` which requires a sync result).